packages/javascript-sdk/src/fr-webauthn/index.ts

```diff
@@ -279,8 +279,8 @@ abstract class FRWebAuthn {
   public static getAuthenticationOutcome(
     credential: PublicKeyCredential | null,
   ):
-    | OutcomeWithName<string, AttestationType, typeof credential>
-    | OutcomeWithName<string, AttestationType, typeof credential, string> {
+    | OutcomeWithName<string, AttestationType, PublicKeyCredential>
```

Comment:
This is a type fix that i errored while doing token vault. can do a beta release with this